### PR TITLE
[zh-cn] Update fundamental_layout_comprehension

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Mainly for the sidebar title, but also for whole page update.

### CODE REVIEW

File: files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md [hunk 1]
```diff
@@ -42,17 +47,35 @@ slug: Learn/CSS/CSS_layout/Fundamental_Layout_Comprehension
...
 
-你能够用好几种方法来实现这些目标，完成一件事的方法并不只有一个。尝试不同的方式，并且看看哪个最有效。尝试之后写下笔记，你可以在本练习的帖子中或者 [#mdn](irc://irc.mozilla.org/mdn) IRC 频道讨论你实现它们的方式。
+1. 将你的作业放在在线可分享编辑器中，如 [CodePen](https://codepen.io/)、[jsFiddle](https://jsfiddle.net) 或 [Glitch](https://glitch.com/)
+2. 在 [MDN Discouse 论坛 Learning 分区](https://discourse.mozilla.org/c/mdn/learn/250)发贴，你的贴文应包含：
 
-## 评估
+  - 一个描述性的标题，如“Assessment wanted for Fundamental layout comprehension”。
+  - 你已经尝试过的细节，以及你希望我们做什么；例如，告诉我们你是否被卡住了，需要帮助或希望进行评估。
+  - 在一个在线可共享的编辑器中，提供你想要评估或需要帮助的例子的链接（在上面第 1 步中提到的）。这是一个很好的做法，如果你不能看到他们的代码，就很难帮助别人解决编码问题。
+  - 实际任务或评估页面的链接，以便我们能够找到你想要帮助的问题。
```
Comment (jasonren0403) [inline files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md:79]:
```suggestion
   - 一个描述性的标题，如“Assessment wanted for Fundamental layout comprehension”。
   - 你已经尝试过的细节，以及你希望我们做什么；例如，告诉我们你是否被卡住了，需要帮助或希望进行评估。
   - 在一个在线可共享的编辑器中，提供你想要评估或需要帮助的例子的链接（在上面第 1 步中提到的）。这是一个很好的做法，如果你不能看到他们的代码，就很难帮助别人解决编码问题。
   - 实际任务或评估页面的链接，以便我们能够找到你想要帮助的问题。
```

File: files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md [hunk 2]
```diff
@@ -1,38 +1,43 @@
...
-如果您已经完成了这个模块的学习，那么您将已经学习了今天进行 CSS 布局以及使用旧的 CSS 所需的基本知识。
+如果你已经完成了本模块的学习，那么你就已经掌握了今天进行 CSS 布局所需要的基本知识，并且也可以使用旧的 CSS。这个任务将通过使用各种技术开发一个简单的网页布局来测试你的一些知识。
 
-<table class="learn-box standard-table">
+<table>
   <tbody>
     <tr>
-      <th scope="row">任务需知：</th>
-      <td><p>在尝试此评估之前，您应该已经阅读了本模块中的所有文章。</p></td>
+      <th scope="row">前提：</th>
```
Comment (yin1999) [inline files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md:14]: 同另一个PR

Comment (jasonren0403) [inline files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md:14]:
```suggestion
      <th scope="row">预备知识：</th>
```

File: files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md [hunk 3]
```diff
@@ -1,38 +1,43 @@
...
+      <td><p>在尝试此评估之前，你应该已经阅读了本模块中的所有文章。</p></td>
     </tr>
     <tr>
-      <th scope="row">任务目标：</th>
-      <td><p>测试您对本单元所涵盖的基本布局技能的理解与掌握应用。</p></td>
+      <th scope="row">目标：</th>
```
Comment (jasonren0403) [inline files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md:18]:
```suggestion
      <th scope="row">学习目标：</th>
```

File: files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md (hunk 2 above)
Comment (jasonren0403) [reply files/zh-cn/learn/css/css_layout/fundamental_layout_comprehension/index.md:14]:
```suggestion
      <th scope="row">预备条件：</th>
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**